src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,110 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 1 << 4;
+    private int capacity;
```

Comment:
don't need this field

Reply:
I use these fields to avoid using a magic number during initialization.
Capacity field is used.

Reply:
Yeah, it's used, but your code would be better off refactored without it. Capacity is basically a (.length) field of another field (table), we can get it any time, by having it we only introduce room for error while updating its value, in this sense just getting length of table is safer and easier. Same for threshold. we read the value of this field only once (but set its value twice), not worth it to have it as a field, just replace it with LOAD_FACTOR * table.length

Reply:
ok, got it